package.json

```
@@ -1,10 +1,10 @@
 {
   "name": "npm-check",
-  "version": "5.9.2",
+  "version": "6.0.0",
   "description": "Check for outdated, incorrect, and unused dependencies.",
   "main": "lib",
   "engines": {
-    "node": ">=0.11.0"
+    "node": ">=10.0.0"
```

Potentially bump this to Node.js 10.9.0 to get V8 6.8. Otherwise we should probably to add `10.0.0` exactly to the test matrix to make sure that we don't accidentally use something introduced there...

```suggestion
    "node": ">=10.9.0"
```